Client/Frontend/Settings/BraveCoreDebugSwitchesView.swift

```diff
@@ -16,20 +16,12 @@ extension BraveCoreSwitch {
       return "Component Updater"
     case .syncURL:
       return "Sync URL"
-    case .skusEnvironment:
```

Comment:
This part should also be moved to a separate PR as reference for 
https://github.com/brave/brave-ios/issues/5643